elixir/apps/web/test/web/live/actors/show_test.exs

```diff
@@ -765,67 +765,21 @@ defmodule Web.Live.Actors.ShowTest do
              }
     end
 
-    test "renders actor identities", %{
+    test "does not render renders actor identities", %{
       account: account,
       actor: actor,
       identity: admin_identity,
       conn: conn
     } do
-      provider = Fixtures.Auth.create_token_provider(account: account)
-
-      identity =
-        Fixtures.Auth.create_identity(
-          account: account,
-          provider: provider,
-          actor: actor,
-          provider_virtual_state: %{
-            "expires_at" => DateTime.utc_now() |> DateTime.add(2, :second)
-          }
-        )
-
       {:ok, lv, _html} =
         conn
         |> authorize_conn(admin_identity)
         |> live(~p"/#{account}/actors/#{actor}")
 
-      lv
-      |> element("#actors")
-      |> render()
-      |> table_to_map()
-      |> with_table_row("identity", "#{provider.name} #{identity.provider_identifier}", fn row ->
-        assert row["actions"] == "Delete"
-        assert around_now?(row["created"])
-        assert row["last signed in"] == "never"
-      end)
+      refute has_element?(lv, "#actors")
```

Comment:
This is a very minor nit, but I just happened to look at the page because of this test....Should the table on the page have an `id` called `identities` rather than `actors` in this case?

Reply:
Good catch